config/database.yml

```diff
@@ -2,14 +2,14 @@ development:
   adapter: postgresql
   database: myrecipes_development
   username: postgres
-  password: <%= Rails.application.credentials.pg[:db_password] %>
+  password: 220690
```

Comment:
should not be hardcoding your password here. Go back to the previous way the password was in there using credentials. See the Readme file on how to set your credentials.

Reply:
I'll check it out!

Reply:
I'll check it out!

Reply:
done!